### PR TITLE
terra-functional-testing and node 12

### LINKED ISSUE #326: Upgrade to use terra-functional-testing
# Story

## Description

As a developer of terra I want to upgrade terra-dev-site testing infrastructure to use [terra-functional-testing](https://github.com/cerner/terra-toolkit/tree/main/packages/terra-functional-testing).

## Motivation

So that the project is on the latest version of terra's WDIO 6 testing infrastructure.

## Definition of Done

We'll know we're done when:

- A PR is open to main referencing the snapshot of terra-functional-testing
- All tests are passing

### LINKED ISSUE #332: Node 12 Upgrade
# Story

## Description

As a maintainer I would like to upgrade this component to Node 14 (lts/fermium)

## Motivation

So that it can have the latest features and security updates and remain tech current with Node.

## Definition of Done

We know we're done when:

- [ ] The .nvmrc file is using lts/fermium
- [ ] All tests are passing
- [ ] A PR is merged to the main branch

## Additional Details
The recently released [version 7](https://webdriver.io/blog/2021/02/09/webdriverio-v7-released) of WebDriverIO dropped support for Node 10 (Dubnium) and recommends to be on Node 12 (Erbium) or higher. The [support|https://github.com/nodejs/release#release-schedule]] for Node 10 will end on 04/30/2021. Upgrading to Node 12 is a prerequisite and will prepare the upgrade to WebDriverIO v7.

### PR DESCRIPTION
**Pull request by jmsv6d**
### Summary

Update to use terra-functional-testing and node 12

Closes #326 
Closes #332 

### Deployment Link

https://terra-dev-site-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: wdio.conf.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-const defaultWdioConfig = require('terra-toolkit/config/wdio/wdio.conf');
+const { config } = require('@cerner/terra-functional-testing');
 
-const wdioConfig = defaultWdioConfig.config;
+const wdioConfig = config;
 
 const travis = process.env.TRAVIS;
```
Comment (mjhenkes) [inline wdio.conf.js:6]: This travis stuff in the wdio config can be removed with corresponding changes to the travis.yml file.

**Review by mjhenkes — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -81,20 +81,21 @@
...
+    "start-heroku": "terra express-server --port $PORT --site './build' ",
+    "start-static": "npm run compile:prod && terra express-server --site ./build",
     "test": "npm run lint && npm run jest && npm run wdio",
     "tds:generate-app-config": "node bin/generateAppConfig.js",
-    "wdio-default": "wdio",
-    "wdio-locale": "LOCALE=de wdio",
-    "wdio-lowlight": "THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
+    "wdio-default": "terra wdio",
+    "wdio-locale": "LOCALE=de terra wdio",
```
Comment (benbcai) [inline package.json:91]:
```suggestion
    "wdio-locale": "terra wdio --locales de",
```

File: package.json [hunk 3]
```diff
@@ -81,20 +81,21 @@
...
+    "wdio-lowlight": "THEME=clinical-lowlight-theme terra wdio",
```
Comment (benbcai) [inline package.json:92]:
```suggestion
    "wdio-lowlight": "terra wdio --themes clinical-lowlight-theme",
```

File: package.json [hunk 4]
```diff
@@ -81,20 +81,21 @@
...
+    "wdio-fusion": "THEME=orion-fusion-theme terra wdio",
```
Comment (benbcai) [inline package.json:93]:
```suggestion
    "wdio-fusion": "terra wdio --themes orion-fusion-theme",
```

File: package.json [hunk 5]
```diff
@@ -178,17 +179,18 @@
     "@babel/preset-env": "^7.5.0",
     "@babel/preset-react": "^7.0.0",
     "@cerner/browserslist-config-terra": "^3.1.0",
-    "@cerner/eslint-config-terra": "^4.0.0",
+    "@cerner/eslint-config-terra": "^5.0.0",
     "@cerner/terra-cli": "^1.0.0",
     "@cerner/terra-open-source-scripts": "^1.0.1",
+    "@cerner/terra-functional-testing": "1.6.0",
```
Comment (benbcai) [inline package.json:185]:
```suggestion
    "@cerner/terra-functional-testing": "^1.6.0",
```

**Comment by benbcai** [comment]
Looks like all of these [error screenshots](https://github.com/cerner/terra-dev-site/tree/terra-functional-testing/tests/wdio/__snapshots__/error/terra-default-theme/de/chrome_undefined) should be deleted. Perhaps add `**/wdio/__snapshots__/error` in .[gitignore](https://github.com/cerner/terra-dev-site/blob/terra-functional-testing/.gitignore#L54-L55).

**Review by yuderekyu — APPROVED** [review]

File: package.json [hunk 6]
```diff
@@ -81,20 +81,21 @@
...
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
+    "wdio-default": "terra wdio",
+    "wdio-locale": "terra wdio --locales de",
+    "wdio-lowlight": "terra wdio --themes clinical-lowlight-theme",
+    "wdio-fusion": "terra wdio --themes orion-fusion-theme",
     "wdio": "npm run wdio-default && npm run wdio-locale && npm run wdio-lowlight && npm run wdio-fusion"
   },
   "dependencies": {
     "@cerner/duplicate-package-checker-webpack-plugin": "^1.0.0",
+    "@cerner/terra-aggregate-translations": "2.0.1",
```
Comment (benbcai) [inline package.json:98]:
```suggestion
    "@cerner/terra-aggregate-translations": "^2.0.1",
```

**Pull request merged**